Task_2/Task_2.cpp

```diff
@@ -1,6 +1,66 @@
 ﻿#include <iostream>
 
+using namespace std;
+
+//РАЗРЯДЫ. Разработайте функцию, которая "переворачивает" переданное ей целое число, например: 1234 -> 4321
+
+int enter(int number);
+int flipPositive(int number);
+int flipNegative(int number);
+
 int main()
 {
-    std::cout << "Hello World!\n";
+    int number = 0, inverted = 0;
+
+
+    number = enter(number);
+
+    if (number >= 0)
+    {
+        inverted = flipPositive(number);
+    }
+    else
+    {
+        inverted = flipNegative(number);
+    }
+
+    cout << "\nInverted number: " << inverted << endl;
+
+    return 0;
+}
+
+int enter(int number)
+{
+    cout << "Enter number: ";
+    cin >> number;
+
+    return number;
+}
+
+int flipPositive(int number)
+{
+    int flip = 0;
+
+    while (number > 0)
+    {
+        flip = (flip * 10) + (number % 10);
+
+        number /= 10;
+    }
+
+    return flip;
+}
+
+int flipNegative(int number)
+{
+    int flip = 0;
+
+    while (number < 0)
+    {
+        flip = (flip * 10) + (number % 10);
+
+        number /= 10;
+    }
+
+    return flip;
```

Comment:
Why?
This is the same functions

Reply:
Refactoring

Reply:
![image](https://user-images.githubusercontent.com/71211228/111910937-7218cf00-8a74-11eb-8bb1-4247bfa00449.png)